mathesar/tests/api/test_schema_api.py

```diff
@@ -49,6 +50,7 @@ def test_schema_list(client, patent_schema, MOD_engine_cache):
     )
 
 
+@pytest.mark.skip("Faulty DB handling assumptions; invalid")
```

Comment:
Please create an issue to fix and re-enable these at some point, might be a good `help wanted` issue.

Reply:
We need to rework the whole testing methodology around DBs. These tests being invalid is a symptom of an underlying problem. I'll make an issue for that, though.